packages/components/docs/token-table.tsx

```diff
@@ -11,23 +14,9 @@ type TokenTableProps = {
   renderExample: (name: string, value: string) => JSX.Element;
 };
 
-const StyledTable = styled.table`
-  border-collapse: separate;
-  border-spacing: 3em 0.5em;
-  text-align: left;
-
-  tbody th {
-    font-weight: normal;
-  }
-
-  td {
-    font-family: monospace;
-  }
-`;
-
 const TokenTable = ({ tokens, renderExample }: TokenTableProps) => {
   return (
-    <StyledTable>
+    <table className={cx("token-table")}>
```

Comment:
Mentioned this in the RFC, but what's the benefit of using `bind` over directly using the class?

```suggestion
    <table className={styles['token-table']}>
```

The values of `styles` are class names, and I'm not aware of any downsides of using them as such.

Reply:
No benefit here, but for the more complex components it's a nice bit of syntactic sugar for concatenating an unknown number of classes together.